<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CELSSearchMainWindow</class>
 <widget class="QMainWindow" name="CELSSearchMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KJPBS ELS/FLS Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblWords">
        <property name="text">
         <string>&amp;Words:</string>
        </property>
        <property name="buddy">
         <cstring>editWords</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editWords"/>
      </item>
      <item>
       <widget class="QLabel" name="lblMinSkip">
        <property name="text">
         <string>Mi&amp;nSkip:</string>
        </property>
        <property name="buddy">
         <cstring>spinMinSkip</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinMinSkip">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblMaxSkip">
        <property name="text">
         <string>Ma&amp;xSkip:</string>
        </property>
        <property name="buddy">
         <cstring>spanMaxSkip</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spanMaxSkip">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSearch">
        <property name="text">
         <string>&amp;Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblWidth">
        <property name="text">
         <string>Wi&amp;dth:</string>
        </property>
        <property name="buddy">
         <cstring>spinWidth</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinWidth">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QTableView" name="tvLetterMatrix">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>10</horstretch>
        <verstretch>10</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="gridStyle">
       <enum>Qt::DashLine</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="btnClear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&amp;Clear</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTableView" name="tvELSResults">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="editSearchLog">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>editWords</tabstop>
  <tabstop>spinMinSkip</tabstop>
  <tabstop>spanMaxSkip</tabstop>
  <tabstop>btnSearch</tabstop>
  <tabstop>tvLetterMatrix</tabstop>
  <tabstop>spinWidth</tabstop>
  <tabstop>tvELSResults</tabstop>
  <tabstop>btnClear</tabstop>
  <tabstop>editSearchLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
